scripts/requirements.txt

```diff
@@ -1,4 +1,4 @@
-conda-build ==2.0.0
+conda-build ==2.0.4
 conda ==4.1.11
```

Comment:
Please don't do this, we need to pin the version and test various things before increasing the version.
Thanks.